storops_test/unity/resource/test_system.py

```diff
@@ -329,6 +331,40 @@ def test_create_nas_server(self):
         nas_server = unity.create_nas_server('nas3', sp, pool)
         assert_that(nas_server.existed, equal_to(True))
 
+    @patch_rest
+    def test_get_user_quota_all(self):
+        unity = t_unity()
+        user_quota_list = unity.get_user_quota()
+        assert_that(user_quota_list, instance_of(UnityUserQuotaList))
+        assert_that(len(user_quota_list), equal_to(2))
+
+    @patch_rest
+    def test_get_specific_user_quota(self):
+        unity = t_unity()
+        user_quota = unity.get_user_quota(_id='userquota_171798692187_3_3')
+        assert_that(user_quota, instance_of(UnityUserQuota))
+        assert_that(user_quota.id, equal_to('userquota_171798692187_3_3'))
+        assert_that(user_quota.existed, equal_to(True))
+
+    @patch_rest
+    def test_create_user_quota(self):
```

Comment:
Could you add some negative test cases, like create an existed user quota, modify a non-existed user quota?

Reply:
@yong-huang done